book/src/dev/proptests.md

```diff
@@ -4,6 +4,13 @@ Zebra uses the [proptest](https://docs.rs/proptest/) crate for randomised proper
 
 Most types in `zebra-chain` have an `Arbitrary` implementation, which generates randomised test cases.
 
+We try ti derive `Arbitrary` impls whenever possible, so that they autoamtically update when we make structural changes.
```

Comment:
```suggestion
We try to derive `Arbitrary` impls whenever possible, so that they automatically update when we make structural changes.
```

Reply:
Thanks for catching this, sometimes I don't notice typos.

Reply:
it was a coincidence that I happened to decide to read the PR changes :D